### ISSUE TITLE
Support for Swift Package Manager (SPM) is broken

### ISSUE THREAD

**Issue by heyzooi** [smartystreets/smartystreets-ios-sdk#15]
Xcode fails in the last step because there's no `Package.swift` file present in the root folder of the repo.

```
git@github.com:smartystreets/smartystreets-ios-sdk.git has no Package.swift manifest for version 8.4.1
```

Here's the pull request for the fix #13

**Comment by DuncanBeutler** [comment]
We're working on fixes! Thanks for raising this issue!

**Comment by heyzooi** [comment]
Here's a fix for it #13

> We're working on fixes! Thanks for raising this issue!

**Comment by DuncanBeutler** [comment]
Pull Request has been reviewed and merged to master. Thank you for contributing!
